src/PassphraseServiceProvider.php

```
@@ -4,42 +4,65 @@
     public function register(): void
     {
         $this->mergeConfigFrom(__DIR__.'/../config/passphrase.php', 'passphrase');
 
-        $this->app->singleton(WordList::class, function (): \NicoBleiler\Passphrase\WordList {
+        $this->app->singleton(WordList::class, function (): WordList {
             $wordList = config('passphrase.word_list');
             $excludedWords = config('passphrase.excluded_words', []);
 
             if (! is_array($excludedWords)) {
-                throw WordListException::invalidExcludedWordsConfigType();
+                throw ConfigException::invalidExcludedWords();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\NicoBleiler\Passphrase\Exceptions\ConfigException' in method 'register'. (undefined)

(StaticAccess)